cfgov/v1/util/__init__.py

```diff
@@ -0,0 +1 @@
+from .util import *
```

Comment:
oh just personal preference, to begin the practice of keeping all of our helper/util classes/methods in one directory. Utils files usually end up growing in time so wanted it to mimic our models structure